src/imports/presets.ts

```diff
@@ -0,0 +1,19 @@
+import type { ModuleOptions } from "./../types";
+import type { InlinePreset } from "unimport";
+import { defineUnimportPreset } from "unimport";
+
+export const createPresets = ({
```

Comment:
### Question

Is [`${outdir}/patterns`](https://panda-css.com/docs/concepts/patterns) not eligible for Auto-imports?

Reply:
We will also support `patttern`!
(It is listed in the todo of the PR.)

Reply:
Oh, sorry. I overlooked it🙏

Reply:
I was stuck at work…
 I really appreciate you making the PR 🙏